uboot-envtools: Add support for Orange Pi R1 Plus & LTS

Vyacguru opened the pull request:
Add support this boards to envtools config
This changes integrates the latest changes from new U-Boot, which includes important updates to the DTSI files for the Orange Pi R1 Plus and Orange Pi R1 Plus LTS boards.

Pugemon commented:
@1715173329 I splitted PR #16034 to one commit with only envtools changes
Please check it

1715173329 commented on package/boot/uboot-envtools/files/rockchip:
```diff
@@ -0,0 +1,23 @@
+#
```
this file needs to convert to use LF line breaks, and add new line at the end of file.
would be better to name as `rockchip_armv8`.

otherwise lgtm.

Vyacguru committed:
uboot-envtools: Add support for Orange Pi R1 Plus & LTS

Add support this boards to envtools config
This commit integrates the latest changes from new U-Boot, which includes important updates to the DTSI files for the Orange Pi R1 Plus and Orange Pi R1 Plus LTS boards.

Signed-off-by: Vyacheslav Ivanov <islavaivanov76@gmail.com>

Vyacguru committed:
Merge branch 'openwrt:main' into opi-r1-plus-envtool
